claudia-lab/lib/response.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.sendJSON = function(res, status, data){
```

Comment:
@loomnugget good use of multiple params for passing in the response object, path, and returned data